### PR TITLE
Add ingest command for composite plots

### PR DESCRIPTION
**Pull request by cjh1**

### CODE REVIEW

File: cli/edp/cli.py [hunk 1]
```diff
@@ -132,3 +134,35 @@ def _ingest(project, cycle, api_url, api_key, dir):
...
+        'name': composite_name
+    }
+    composite = gc.post('edp/projects/%s/composites' % project, json=composite)
+    composite = composite['_id']
+
+    channel_map = json.load(channel_map)
+    channel_map = {channel.upper():element.lower() for (channel,element) in channel_map.items()}
+
+    experiments = _ingest_runs(gc, project, composite, dir)
+    #(run_ids, runs) = _ingest_runs(gc, project, composite, dir)
```
Comment (alesgenova) [inline cli/edp/cli.py]: Unused code

File: cli/edp/composite/__init__.py [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+            technique_files = {}
+            for key, value in run.items():
+                if key.startswith('files_technique__'):
+                    match = technque_file_regex.match(key)
+                    technique = match.group(1)
+
+                    run_dir = os.path.dirname(run_file)
+                    technique_files[technique] = [os.path.join(run_dir, f) for f in value['pstat_files'].keys()]
+
+                #if key.startswith('echem_params'):
```
Comment (alesgenova) [inline cli/edp/composite/__init__.py]: Leftover debug

Comment (cjh1) [reply cli/edp/composite/__init__.py]: Done

File: cli/edp/cli.py (hunk 1 above)
Comment (cjh1) [reply cli/edp/cli.py]: Done

**Pull request merged**